docs/man/man1/oadm-policy-scc-review.1

```diff
@@ -1,3 +0,0 @@
-This file is autogenerated, but we've stopped checking such files into the
```

Comment:
ah, I see - some issue with my local build environment - using hack/env seems to have fixed this, thanks